gdc-viewer/js/Store/SeqFeature/BAM.js

```diff
@@ -0,0 +1,277 @@
+const LRU = cjsRequire('quick-lru')
+const { BamFile } = cjsRequire('@gmod/bam')
+
+
+const bamIndexedFilesCache = new LRU({ maxSize: 5 })
+
+const BlobFilehandleWrapper = cjsRequire('../../Model/BlobFilehandleWrapper')
```

Comment:
This should point to `JBrowse/Model/BlobFilehandleWrapper` - no need to duplicate this file